dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/managedobject/CDIManagedObjectService.java

```diff
@@ -36,57 +36,31 @@
                                                                                                                                                         "service.ranking:Integer=9999" })
 public class CDIManagedObjectService implements ManagedObjectService {
 
-    private final AtomicServiceReference<CDIService> cdiServiceRef = new AtomicServiceReference<CDIService>("cdiService");
-    private final AtomicServiceReference<DefaultManagedObjectService> defaultMOSRef = new AtomicServiceReference<DefaultManagedObjectService>("defaultManagedObjectService");
+    private DefaultManagedObjectService defaultMOS;
+    private CDIService cdiService;
 
     private CDIRuntime cdiRuntime;
 
     public void activate(ComponentContext cc) {
-        cdiServiceRef.activate(cc);
-        defaultMOSRef.activate(cc);
     }
 
     public void deactivate(ComponentContext cc) {
-        cdiServiceRef.deactivate(cc);
-        defaultMOSRef.deactivate(cc);
         cdiRuntime = null;
```

Comment:
There is no need to null out this mandatory reference on deactivate.  I suggest deleting this whole method.